fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java

```diff
@@ -169,13 +181,15 @@ public static List<Column> genColumnsFromEs(EsRestClient client, String indexNam
             boolean mappingEsId) {
         String mapping = client.getMapping(indexName);
         ObjectNode mappings = getMapping(mapping);
-        ObjectNode rootSchema = getRootSchema(mappings, mappingType);
-        return genColumnsFromEs(indexName, mappingType, rootSchema, mappingEsId);
+        // Get array_fields while removing _meta property.
+        List<String> arrayFields = new ArrayList<>();
+        ObjectNode rootSchema = getRootSchema(mappings, mappingType, arrayFields);
```

Comment:
This methon is a bit like C++ style. How about this?
```suggestion
        List<String> arrayFields = extractArrayFields(mappings, mappingType);
        ObjectNode rootSchema = getRootSchema(mappings, mappingType);
```

Reply:
We can only get `array_fields` during the `getRootSchema` before the `_meta` field is removed.
The suggestion changes are more like the style before the refactor, which will add ES version judgement in the `extractArrayFields` and will be duplicated with the logical in `getRootSchema`.
You can check the old style via git history.